tests/testthat/test-public_api.R

```diff
@@ -21,3 +21,39 @@ test_that("styler does not return error when there is no file to style", {
 })
```

Comment:
Would the preceding tests change the original files?

Reply:
No, they should not, because the input is already styled. The tests are only to check whether the call returns an error or not. It's a rather superficial test.

Reply:
These files will then need an update whenever the style guide changes, but that's fine with me.